hackathon_site/event/serializers.py

```diff
@@ -32,6 +32,15 @@ class Meta:
             "e_signature",
             "team",
         )
+        read_only_fields = ("id", "team")
+
+
+class UserProfileSerializer(ProfileSerializer):
```

Comment:
Minor, would call this `CurrentProfileSerializer` just to be consistent with the view.